whylog/config/parsers.py

```diff
@@ -162,6 +179,13 @@ def _create_group_index_to_regex_name(self, parsers_indexes):
             index_to_regex[indexes[0]] = name
         return index_to_regex
 
+    def get_clues_from_matched_line(self, line):
```

Comment:
If I understand correctly, this function does not returns Clue's yet. If so, the name of this function is a bit confusing. Could you rename it?